.github/workflows/eerelease.yml

```
@@ -0,0 +1,127 @@
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+      packages: write
+
+    steps:
+      - name: Check out images
+        uses: actions/checkout@v4
+        with:
+          repository: ansible-community/images
+          ref: main
+          path: image
+
+      - name: Check out eercheck repo for testing
+        uses: actions/checkout@v4
+        with:
+          repository: anweshadas/eercheck
+          ref: main
+          path: eercheck
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install ansible-builder
+          python3 -m pip install setuptools
+          sudo apt install podman --yes
+          pwd
+          ls -l
+
+      - name: Pull the latest fedora image
+        run: |
+          podman pull quay.io/fedora/fedora:latest
+
+      - name: Build Community EE image
+        working-directory: image/execution-environments/${{ inputs.ee-name }}
+        run: |
+          ansible-builder build --tag ghcr.io/ansible-community/${{ inputs.ee-name }}:${{ env.EE_VERSION }}
+
+      - name: Find the id of the latest EE image
+        run: |
+          echo "IMAGE_ID=$(podman images -q ${{ inputs.ee-name }}:${{ env.EE_VERSION }})">>$GITHUB_ENV
+
+      - name: Test the image with eercheck
+        working-directory: eercheck
+        run: |
+          pwd
+          ls -l
+          python3 -m venv .venv
+          source .venv/bin/activate
+          python -m pip install -r requirements.txt
+          sudo systemctl start podman.socket
+          ./containertest.py ${{ env.IMAGE_ID }}
+          echo "STATUS_CODE=$?">>$GITHUB_ENV
+
+      - name: Checking the STATUS_CODE
+        run: |
+          if test ${{ env.STATUS_CODE }}!= 0; then
+            exit 1;
+          fi
+
+      - name: Create the latest tag
+        if: ${{ inputs.tag_as_latest }}
+        run: |
+          podman tag ${{ env.IMAGE_ID }} ghcr.io/ansible-community/${{ inputs.ee-name }}:latest
+          podman images
+
+      - name: Login to the Github Container registry
+        env:
+          REGISTRY_USER: ${{ github.actor }}
+        uses: redhat-actions/podman-login@v1
+        with:
+          registry: ghcr.io/${{ github.repository_owner }}
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+
+      - name: Upload artifact EE to ghcr
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ inputs.ee-name }}
+          tags: ${{ env.EE_VERSION }}
+          registry: ghcr.io/ansible-community
+
+      - name: Check and upload the image latest tag
+        if: ${{ inputs.tag_as_latest}}
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ inputs.ee-name }}
+          tags: latest
+          registry: ghcr.io/ansible-community
```

Shouldn't either this be
```suggestion
          registry: ghcr.io/${{ github.repository_owner }}
```
as well, or the one above be changed to the hard-coded owner name?

---

Updated to this `registry: ghcr.io/${{ github.repository_owner }}` .